packages/core/saved-objects/docs/openapi/bundled.yaml

```diff
@@ -62,6 +207,57 @@ paths:
         - url: https://localhost:5601
     servers:
       - url: https://localhost:5601
+  /api/encrypted_saved_objects/_rotate_key:
+    post:
+      summary: Rotate the encryption key for encrypted saved objects.
+      operationId: rotateEncryptionKey
+      description: |
+        Superuser role required. This functionality is in technical preview and may be changed or removed in a future release. Elastic will apply best effort to fix any issues, but features in technical preview are not subject to the support SLA of official GA features. NOTE: In most cases, total will be greater than successful even if failed is zero. The reason is that Kibana may not need or may not be able to rotate encryption keys for all encrypted saved objects. 
+      tags:
+        - saved objects
+      parameters:
```

Comment:
In the [docs](https://www.elastic.co/guide/en/kibana/current/saved-objects-api-rotate-encryption-key.html#saved-objects-api-rotate-encryption-key-request-query-params) I see 2 query params, `type` and `batch_size`. The other sections list the query params in the `parameters` section, these are being listed under `Request Body >Schema`. Should they be moved to match the other sections

Reply:
Thanks for catching this. Got this fixed up now:

<img width="746" alt="Screenshot 2023-08-23 at 5 17 36 PM" src="https://github.com/elastic/kibana/assets/40268737/67d52eb9-cdf1-42db-ad37-3c6d79b9a276">